packages/inscription-protocol/src/defaults/common-defaults.ts

```diff
@@ -115,7 +115,7 @@ export const DEFAULT_REQUEST_DATA: RequestData = {
   },
   permission: {
     anonymous: true,
-    role: 'Everybody',
+    roles: ['Everybody'] as string[],
```

Comment:
this as string[] should not be needed right?

Reply:
not needed. other string arrays are also defined this way in this file.

Reply:
needed if they are initalized empty, but not if they have a value. but not wrong